### PR TITLE
Add Validator Community Page

### PR DESCRIPTION
**Pull request by ansonla3**

### CODE REVIEW

File: website/i18n/en.json [hunk 1]
```diff
@@ -143,7 +143,6 @@
       },
```
Comment (krichard410) [inline website/i18n/en.json]: Could you remove the `website/i18n/en.json` file from this PR? When it gets pushed up to master, it gets generated.

File: docs/maintain-guides-how-to-build-validator-community.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+Validators are welcome to promote your node in the channels listed below or other social media
+platforms to let more people know about what is about your validator, what have you done for the
+network, why nominators should support your work. There are multiple ways to help you to get
+nominations from the community, for example, write a staking tutorial in a specific language that
+targets those communities, provide staking tooling to the community, etc..
+
+## Channels
+
+Riot: [Polkadot Validator Lounge](https://riot.im/app/#/room/#polkadot-validator-lounge:matrix.org)
```
Comment (krichard410) [inline docs/maintain-guides-how-to-build-validator-community.md]: I'm wondering if it'd be appropriate to also list Telegram in this as well.

File: docs/maintain-guides-how-to-build-validator-community.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+title: Build a Community for Your Validator
+sidebar_label: Build a Community for Your Validator
+description: Build a Community for Your Validator
+---
+
+Validators are welcome to promote your node in the channels listed below or other social media
+platforms to let more people know about what is about your validator, what have you done for the
+network, why nominators should support your work. There are multiple ways to help you to get
+nominations from the community, for example, write a staking tutorial in a specific language that
+targets those communities, provide staking tooling to the community, etc..
```
Comment (krichard410) [inline docs/maintain-guides-how-to-build-validator-community.md]: I would reword this part as a list. So for example:
`There are multiple ways to obtain nominations from the community, some of them being:
- list 1 
- list 2
`

File: docs/maintain-guides-how-to-build-validator-community.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+---
+id: maintain-guides-how-to-build-validator-community
+title: Build a Community for Your Validator
+sidebar_label: Build a Community for Your Validator
+description: Build a Community for Your Validator
+---
+
+Validators are welcome to promote your node in the channels listed below or other social media
```
Comment (krichard410) [inline docs/maintain-guides-how-to-build-validator-community.md]:
```suggestion
Validators are welcome to promote their node in the channels listed below or other social media
```

File: docs/maintain-guides-how-to-build-validator-community.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+platforms to let more people know about what is about your validator, what have you done for the
```
Comment (krichard410) [inline docs/maintain-guides-how-to-build-validator-community.md]:
```suggestion
platforms to let more people know about your validator stats, what have you done for the
```

File: docs/maintain-guides-how-to-build-validator-community.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+network, why nominators should support your work. There are multiple ways to help you to get
```
Comment (krichard410) [inline docs/maintain-guides-how-to-build-validator-community.md]:
```suggestion
network, and why nominators should support your work. There are multiple ways to help you to get
```
I'm not sure if you need both `what you have done for the network` and `validator stats`, are they the same thing?

File: docs/maintain-guides-how-to-build-validator-community.md (hunk 3 above)
Comment (krichard410) [reply docs/maintain-guides-how-to-build-validator-community.md]: I'd also recommend here that another idea is to answer peoples questions they have on social media accounts. The more people recognize your name, the more they might be willing to nominate for you.

File: docs/maintain-guides-how-to-build-validator-community.md (hunk 4 above)
Comment (lsaether) [inline docs/maintain-guides-how-to-build-validator-community.md]:
```suggestion
Validators are welcome to promote your service in the channels listed below or other social media
```

File: docs/maintain-guides-how-to-build-validator-community.md (hunk 3 above)
Comment (lsaether) [inline docs/maintain-guides-how-to-build-validator-community.md]:
```suggestion
targets those communities, provide staking tooling to the community, etc.
```

File: docs/maintain-guides-how-to-build-validator-community.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+nominations from the community, for example, write a staking tutorial in a specific language that
+targets those communities, provide staking tooling to the community, etc..
+
+## Channels
```
Comment (lsaether) [inline docs/maintain-guides-how-to-build-validator-community.md]: Can we change this into a bullet point list? It just seems cleaner than making each link a separate line.

File: docs/maintain-guides-how-to-build-validator-community.md [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+title: Build a Community for Your Validator
+sidebar_label: Build a Community for Your Validator
+description: Build a Community for Your Validator
+---
+
+Validators are welcome to promote their node in the channels listed below or other social media
+platforms to let more people know about your validator stats, what have you done for the network,
+and why nominators should support your work. There are multiple ways to help you to get nominations
+from the community, for example, write a staking tutorial in a specific language that targets those
+communities, provide staking tooling to the community, etc.
```
Comment (Swader) [inline docs/maintain-guides-how-to-build-validator-community.md]:
```suggestion
Validators are welcome to promote their node in the channels listed below or other social media
platforms to let more people know about their validator stats, what they have done for the network,
and why nominators should support their work. There are multiple ways to help you to get nominations
from the community For example, write a staking tutorial in a specific language that targets those
communities, provide staking tooling to the community, etc.
```

**Review by Swader — CHANGES_REQUESTED** [review]

File: docs/maintain-guides-how-to-build-validator-community.md [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+id: maintain-guides-how-to-build-validator-community
+title: Build a Community for Your Validator
+sidebar_label: Build a Community for Your Validator
+description: Build a Community for Your Validator
+---
+
+Validators are welcome to promote their node in the channels listed below or other social media
+platforms to let more people know about their validator stats, what they have done for the network,
+and why nominators should support their work. There are multiple ways to help you to get nominations
+from the community For example, write a staking tutorial in a specific language that targets those
```
Comment (krichard410) [inline docs/maintain-guides-how-to-build-validator-community.md]:
```suggestion
from the community. For example, write a staking tutorial in a specific language that targets those
```

File: docs/maintain-guides-how-to-build-validator-community.md [hunk 10]
```diff
@@ -0,0 +1,20 @@
...
+communities, provide staking tooling to the community, etc.
```
Comment (krichard410) [inline docs/maintain-guides-how-to-build-validator-community.md]:
```suggestion
communities, provide staking tooling to the community, helping others with understanding Polkadot on Telegram or Element, etc.
```

File: docs/maintain-guides-how-to-build-validator-community.md [hunk 11]
```diff
@@ -0,0 +1,20 @@
...
+
+## Channels
+
+Riot: [Polkadot Validator Lounge](https://riot.im/app/#/room/#polkadot-validator-lounge:matrix.org)
```
Comment (krichard410) [inline docs/maintain-guides-how-to-build-validator-community.md]: It's called Element now =)

**Review by Swader — APPROVED** [review]

File: docs/maintain-guides-how-to-build-validator-community.md [hunk 12]
```diff
@@ -0,0 +1,21 @@
+---
```
Comment (krichard410) [inline docs/maintain-guides-how-to-build-validator-community.md:1]: I'm wondering if this would make sense to have this inside another page rather than a separate page itself? Since the material is small, we could place it inside learn-validator or another page for instance. @Swader @lsaether @ansonla3 what do you think?

Comment (Swader) [reply docs/maintain-guides-how-to-build-validator-community.md:1]: Agreed, this should be a part of the how to be a validator page. We have too many pages in the wiki.

**Comment by lsaether** [comment]
OTOH the validator guide is already pretty long. I would argue that article length bloat is worse than page number bloat. It would be easier to find this content based on the sidebar labels than it would be to find this paragraph in the already long validator guide.  I could also see this page evolving over time to be more comprehensive.

**Comment by Swader** [comment]
Fair

**Comment by laboon** [comment]
What is the status of this?
